### PR TITLE
Add description to services in search view

### PR DESCRIPTION
**Pull request by lukeperrin10**
PT story: https://www.pivotaltracker.com/story/show/179251261

## Suggested changes 

- adds first sentence of main description as preview
- on show more, preview is hidden 
- updates website to be link 
- link opens in new tab

### CODE REVIEW

File: src/components/Service.jsx [hunk 1]
```diff
@@ -46,25 +69,42 @@ const ScreenSplit = ({ data }) => {
...
+            style={styles.contacts}
+            className={classes.text}
+          >
             {data.telephone && `phone: ${data.telephone}\n`}
             {data.email && `email: ${data.email}\n`}
             {data.address && `address: ${data.address}\n`}
             {data.postcode && `postcode: ${data.postcode}\n`}
-            {data.website && `website: ${data.website}\n`}
+            Website:
+            {data.website ? (
```
Comment (SevaDer14) [inline src/components/Service.jsx:102]: change ? for && and then you dont need : undefined

File: src/components/Service.jsx [hunk 2]
```diff
@@ -30,11 +37,27 @@ const ScreenSplit = ({ data }) => {
...
         <Box component="div" style={styles.map}>
-          <Button data-cy="toggle-map-visibility-button" size="small" color="secondary" onClick={()=>setCollapse(true)}>Collapse map</Button>
+          <Button
+            data-cy="toggle-map-visibility-button"
+            size="small"
+            color="secondary"
+            onClick={() => setCollapse(true)}
+          >
+            Collapse map
+          </Button>
```
Comment (SevaDer14) [inline src/components/Service.jsx:60]: It was there before but we can refactor it. replace to `setCollapse(!collapse)`, and button content to ternary `{collapse ? 'Show on map' : 'Collapse map'}`

then we dont need if statement at all

File: src/components/ServiceListItem.jsx [hunk 3]
```diff
@@ -51,12 +54,16 @@ const ServiceListItem = ({ listing, index }) => {
                   component="p"
                   style={styles.shortDescription}
                 >
-                  Can be some short description here...
+                  {!expanded ? `${shortDescriptionText}...` : undefined}
```
Comment (SevaDer14) [inline src/components/ServiceListItem.jsx:57]: also can be && instead of ?

**Review by SevaDer14 — CHANGES_REQUESTED** [review]
Some small stuff

File: src/components/Service.jsx (hunk 1 above)
Comment (lukeperrin10) [reply src/components/Service.jsx:102]: Fixed :)

File: src/components/ServiceListItem.jsx (hunk 3 above)
Comment (lukeperrin10) [reply src/components/ServiceListItem.jsx:57]: Fixed

File: src/components/Service.jsx (hunk 2 above)
Comment (lukeperrin10) [reply src/components/Service.jsx:60]: fixed

**Review by SevaDer14 — APPROVED** [review]
Great job!

**Pull request merged**